sycl/source/detail/config.cpp

```diff
@@ -62,36 +63,70 @@ void readConfig() {
   }
 
   if (File.is_open()) {
-    // TODO: Use max size from macro instead of 256
-    char Key[MAX_CONFIG_NAME] = {0}, Value[256] = {0};
+    char Key[MAX_CONFIG_NAME] = {0}, Value[MAX_CONFIG_VALUE] = {0};
+    std::string BufString;
+    std::size_t Position = std::string::npos;
     while (!File.eof()) {
-      // Expected fromat:
+      // Expected format:
       // ConfigName=Value\r
       // ConfigName=Value
       // TODO: Skip spaces before and after '='
```

Comment:
should we remove the TODO?

Reply:
TODO was added two years ago. I can remove it, nevertheless we still don't skip spaces at the beginning/end and before/after assignment (we throw exception if face such cases).

Reply:
can we fix this TODO and remove it?

Reply:
you can leave it as is.